src/Entities/Aero/AeroBlockSlingshot.cs

```diff
@@ -347,6 +348,9 @@ private void OnPush(int moveX, Pushable.MoveActionType moveAction)
     {
         stateMachine.State = (int) SlingshotStates.Windup;
         updateTimer = SetTime;
+
+        var speed = (pushable.MaxPushSpeed < 0 ? 70f : Math.Min(Player.MaxRun, pushable.MaxPushSpeed)) * Input.MoveX.Value;
```

Comment:
This should use `moveX` rather than `Input.MoveX.Value`, otherwise it's restricted to actual player input.  In the future we may want to other entities to interact with `Pushable`s.

`OnPush` is already called using `Input.MoveX.Value`, but not relying on this outside the component gives us greater flexibility in the future.

Reply:
gotcha. just pushed the change. should've been a little more careful instead of pasting the line there